bin/MadGraph5_aMCatNLO/cards/production/13TeV/HZaTo2l2g/HZaTo2l2g_run_card.dat

```diff
@@ -0,0 +1,270 @@
+#                                                                    *
+#  Some notation/conventions:                                        *
+#                                                                    *
+#   Lines starting with a '# ' are info or comments                  *
+#                                                                    *
+#   mind the format:   value    = variable     ! comment             *
+#*********************************************************************
+#
+#*******************                                                 
+# Running parameters
+#*******************                                                 
+#                                                                    
+#*********************************************************************
+# Tag name for the run (one word)                                    *
+#*********************************************************************
+  tag_1     = run_tag ! name of the run 
+#*********************************************************************
+# Run to generate the grid pack                                      *
+#*********************************************************************
+  .true.     = gridpack  !True = setting up the grid pack
+#*********************************************************************
+# Number of events and rnd seed                                      *
+# Warning: Do not generate more than 1M events in a single run       *
+# If you want to run Pythia, avoid more than 50k events in a run.    *
+#*********************************************************************
+  1000 = nevents ! Number of unweighted events requested 
+      0       = iseed   ! rnd seed (0=assigned automatically=default))
+#*********************************************************************
+# Collider type and energy                                           *
+# lpp: 0=No PDF, 1=proton, -1=antiproton, 2=photon from proton,      *
+#                                         3=photon from electron     *
+#*********************************************************************
+        1     = lpp1    ! beam 1 type 
+        1     = lpp2    ! beam 2 type
+     6500     = ebeam1  ! beam 1 total energy in GeV
+     6500     = ebeam2  ! beam 2 total energy in GeV
+#*********************************************************************
+# Beam polarization from -100 (left-handed) to 100 (right-handed)    *
+#*********************************************************************
+        0     = polbeam1 ! beam polarization for beam 1
+        0     = polbeam2 ! beam polarization for beam 2
+#*********************************************************************
+# PDF CHOICE: this automatically fixes also alpha_s and its evol.    *
+#*********************************************************************
+ 'lhapdf'    = pdlabel     ! PDF set                                  
+ 260000 = lhaid     ! if pdlabel=lhapdf, this is the lhapdf number 
```

Comment:
Is this old pdf used for some reason?

Reply:
Yes this is needed as we would like to produce signal samples consistent with re-reco.

Reply:
But this is NLO pdf : NNPDF30_nlo_as_0118
Should not use LO PDF?

Reply:
For which year are you planning to produce the signal samples though? For 2017 and 2018, NNPDF31 is the recommendation.

Reply:
We are planning to produce all three years. We will change the pdf to NNPDF31 for 2017 and 2018, as already requested by the Higgs MC contact.

Reply:
To be specific we will use 260000 for 2016 and 306000 for 2017-2018.

Reply:
Will this go to UL ? Or pre-legacy?

Reply:
I don't quite get the reasons for using 260000 and 306000, both.
260000:NNPDF30_nlo_as_0118 is an NLO PDF and 306000 is a no longer recommended one (325300 replaced it)